base/ltcmd.dtx

```diff
@@ -97,6 +97,10 @@
 %<latexrelease>                 {Document~command~parser}%
 %    \end{macrocode}
 %
+%    \begin{macrocode}
+\cs_generate_variant:Nn \tl_head:n { e }
+%    \end{macrocode}
```

Comment:
shouldn't this go into l3kernel instead?

Reply:
Yes, and it has today - but I need to allow for older-`expl3`-newer-kernel.